sme_finance_application_schema/entity_v1

```diff
@@ -88,14 +85,13 @@
         },
         "overseas_revenue": {
             "title":"Overseas revenue",
-            "description":"The proportion of revenue the SME receives from sales outside the United Kingdom, expressed as a percentage",
+            "description":"The monthly revenue the SME receives from sales outside the United Kingdom, in GBP",
```

Comment:
uhm, this departs from sme_v5, doesn't it? If that's the case, the sme_v5_and_contact_to_finance_application_v3 is broken